app/src/main/java/cs2063/groceryshopper/util/ListOfItemsGenerator.kt

```diff
@@ -1,23 +1,42 @@
 package cs2063.groceryshopper.util
 
+import android.content.Context
+import android.graphics.Color
 import android.os.Handler
 import android.os.Looper
+import android.view.View
+import android.view.ViewGroup
 import android.widget.ListView
 import android.widget.SimpleAdapter
+import android.widget.Spinner
+import android.widget.TextView
+import androidx.annotation.IdRes
+import androidx.annotation.LayoutRes
 import cs2063.groceryshopper.R
 import cs2063.groceryshopper.TripActivity
 import cs2063.groceryshopper.model.Item
+import java.util.Locale
 import java.util.concurrent.Executors
 
 
 class ListOfItemsGenerator {
     fun generateList(activity : TripActivity, db: DBHelper, tripId: Int) {
+
+        // TODO: BEN -> Might not need to be here?
```

Comment:
Here @bjacobsUNB

Reply:
Im not sure what there is for me to do here. Is this just a question because even if its not doing anything its not gonna hurt it enough to matter